packages/gatsby-source-wordpress/src/steps/declare-plugin-options-schema.ts

```diff
@@ -415,6 +415,56 @@ When using this option, be sure to gitignore the wordpress-cache directory in th
             },
           `),
       }),
+
+    httpOpts: Joi.object({
```

Comment:
Options related to local file nodes should go under `options.type.MediaItem.localFile` https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/plugin-options.md#typemediaitemlocalfile

Reply:
Also can you rename the option to `httpOptions`? The other options for this plugin usually spell out the full word 👍 even though in this case we're passing it down to a different lib where it's called httpOpts I think we should keep option naming consistent.

Reply:
Ok it's done :+1: 

I tried the new config and it worked fine.
I updated the doc manually because when I run `npm run build` for the package, it works and I can use this output as a dependency inside my test project, but Joi doesn't generate the doc, idk why :/  

I tried from master and I have exactly the same issue. Joi output is empty

```diff
diff --git a/packages/gatsby-source-wordpress/docs/plugin-options.md b/packages/gatsby-source-wordpress/docs/plugin-options.md
index 849c0c0..0eb87d2 100644
--- a/packages/gatsby-source-wordpress/docs/plugin-options.md
+++ b/packages/gatsby-source-wordpress/docs/plugin-options.md
@@ -2,1347 +2,6 @@
```